docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -509,16 +577,16 @@ The figure below demonstrates what the Motor tab should look like in QGroundCont
 
     QGroundControl Motor Testing
 
-See the `Successful Test Videos`_ section below for a video of a similar test process using ArduCopter and Mission Planner, which demonstrates what sounds you should expect from the motor and it successfully
+See the `Successful Test Videos`_ section below for a video of a similar test process using ArduCopter and Mission Planner, which demonstrates what sounds you should expect from the module and it successfully
```

Comment:
... and Mission Planner. The video demonstrates what sounds you should expect from the module, and it successfully spinning as controlled by a flight controller.